src/common/cscs_api_common.py

```diff
@@ -337,6 +337,10 @@ def exec_remote_command(auth_header, system_name, system_addr, action, file_tran
         if stderr_errno == 0:
             if stderr_errda and not in_str(stderr_errda,"Could not chdir to home directory"):
                 result = {"error": 0, "msg": stderr_errda}
+            elif in_str(stdout_errda, "No such file"): # in case that error is 0 and the msg is on the stdout (like with some file)
+                result = {"error": 1, "msg": stdout_errda} 
+            elif in_str(stdout_errda, "no read permission"): # in case that error is 0 and the msg is on the stdout (like with some file)
```

Comment:
When I don't have the right permissions for a file the type command returns ```cannot open `filename' (Permission denied)```, am I missing something here? Why are you checking for "no read permission"?

Reply:
`/utilities/file` ndpoint uses `file` command, not `type` 
Outputs for `file` in each case are (file not found or permission error) are:
```
[testuser@cluster ~]$ file -b -- "not_a_file"
cannot open (No such file or directory)
[testuser@cluster ~]$ file -b -- "/var/log/messages"
regular file, no read permission
```

Reply:
Sorry, I also meant the file command, "type" in my comment was a typo. But in the issue I was trying something from a path where I don't have permission (for example a file inside some other user's home). In the case that you mention above the type of the file should return normally without an exception, no? Since the user has permission to see the type of the file (in this case regular file).

Reply:
I see, you mean a file that can't be listed. I will check that case.